docusaurus/docs/components/analytics/analytics.md

```diff
@@ -53,7 +78,7 @@ When an event is created, if `true`, will traverse up the DOM tree from the give
 
 #### `attributePrefix?: string`
 
-Prefix to apply on event attributes you want to stripe off components on click. **Default:** `data-analytics`
+Prefix to apply on event attributes you want to strip off components on click. **Default:** `data-analytics`
```

Comment:
Do we need more detailed documentation on what the attributePrefix does? Seemed like there was confusion there.

Reply:
Sure, I could provide some example code here.

Also, I could not for the life of me figure out how to get `eventModifiers` below to do anything useful. If you have any insights on that, do tell!